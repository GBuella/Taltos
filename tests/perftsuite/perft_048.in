setboard k7/B7/1B6/1B6/8/8/8/K6b w - - 0 1
echo k7/B7/1B6/1B6/8/8/8/K6b w - - 0 1
perft 1
perft 2
perft 3
perft 4
perft 5
perft 6
