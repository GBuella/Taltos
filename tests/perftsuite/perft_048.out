k7/B7/1B6/1B6/8/8/8/K6b w - - 0 1
21
144
3242
32955
787524
7881673
